content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,353 @@
+serviços Windows.
+
+## {{% heading "prerequisites" %}}
+
+Você precisa ter um cluster Kubernetes, e a ferramenta de linha de comando `kubectl` 
+precisa estar configurada para comunicar-se com seu cluster.
+O cluster deve possuir nós de carga de trabalho Windows. 
+Esta seção cobre o conjunto inicial de passos requeridos para cada cluster:
+
+### Instale o CRD GMSACredentialSpec
+
+Uma [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) (CRD) para a especificação de recursos de credencial GMSA precisa ser configurada no cluster, para definir o tipo de recurso do cliente `GMSACredentialSpec`. Faça o download do [YAML](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/gmsa-crd.yml) do CRD de GMSA 
+e salve como gmsa-crd.yaml.
+A seguir, instale o CRD com `kubectl apply -f gmsa-crd.yaml`.
+
+### Instale webhooks para validar usuários GMSA
+
+Dois webhooks precisam ser configurados no cluster Kubernetes para popular e validar 
+as referências de especificação de credenciais GMSA no nível do Pod ou contêiner:
+
+1. Um webhook de mutação que expanda as referências para as GMSAs,
+(por nome a partir de uma especificação de Pod) em uma especificação de credencial completa 
+em formato JSON dentro da especificação do Pod.
+
+1. Um webhook de validação garante que todas as referências para GMSAs estão 
+autorizadas a serem usadas pela conta de serviço do Pod.
+
+A instalação dos webhooks acima e dos objetos associados requer as etapas abaixo: 
+
+1. Crie um par de chaves de certificado (que será usado para permitir que o 
+contêiner do webhook se comunique com o cluster)
+
+1. Instale um Secret com o certificado acima.
+
+1. Crie um Deployment para a lógica principal do webhook.
+
+1. Crie as configurações de webhook de validação e de mutação, referentes ao Deployment.
+
+Um [script](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh) 
+pode ser usado para implantar e configurar os webhooks GMSA e objetos associados 
+mencionados acima. O script pode ser executado com a opção ```--dry-run=server``` 
+para possibilitar que você possa revisar as alterações antes que sejam aplicadas 
+no seu cluster.
+
+O [template YAML](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/gmsa-webhook.yml.tpl) 
+usado pelo script também pode ser usado para implantar os webhooks e objetos 
+associados manualmente (com as substituições apropriadas para os parâmetros).
+
+<!-- steps -->
+
+## Configurar GMSAs e nós Windows em Active Directory
+
+Antes que os Pods no Kubernetes possam ser configurados para usar GMSAs, as GMSAs apropriadas precisam ser provisionadas no Active Directory como descrito na 
+[documentação de GMSA do Windows](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/getting-started-with-group-managed-service-accounts#BKMK_Step1). 
```

Comment:
Minor: use pt-br instead of en-us
(https://docs.microsoft.com/pt-br/windows-server/security/group-managed-service-accounts/getting-started-with-group-managed-service-accounts#BKMK_Step1